src/components/GradeItemAccordion.tsx

```diff
@@ -8,51 +8,98 @@ export interface GradeItemAccordionInterface {
 	updateItem: (item: CourseItem) => void;
 }
 
-const Accordion = styled.tr`
-	background-color: #ffffff;
+const Accordion = styled.div`
 	color: #4f4f4f;
-	padding: 2em;
-	justify-content: center;
-	line-height: 8em;
-	transition: line-height 14em ease-out;
-	border: #ffffff;
-	text-decoration: none;
-	border: none;
+
+	display: grid;
+	grid-template-columns:
+		minmax(75px, 1fr) minmax(200px, 5fr) minmax(200px, 5fr)
+		minmax(200px, 5fr) minmax(200px, 5fr);
+	grid-template-rows: 100px;
+	grid-template-areas: 'arrow item weight grade duedate';
 `;
-const RowComponent = styled.td`
-	border: #ffffff;
+const RowComponent = styled.div`
 	font-weight: bold;
 	font-size: large;
 	border: none;
-	margin: 1.5em;
+
+	background-color: #ffffff;
 `;
 const DropDownArrow = styled.button`
 	border: none;
 	outline: none;
-	background-color: #ffffff;
 	font-size: 2em;
+	background-color: #ffffff;
 `;
-const AccordionExtended = styled.tr`
+const AccordionExtended = styled.div`
 	font-weight: bold;
 	font-size: large;
 	color: #4961e1;
 	border: none;
-`;
-const AccordionExtendedComponent = styled.td`
 	background-color: #ffffff;
+
+	display: grid;
+	grid-template-columns:
+		minmax(75px, 1fr) minmax(200px, 5fr) minmax(200px, 5fr)
+		minmax(200px, 5fr) minmax(200px, 5fr);
+	grid-template-rows: 75px;
+	grid-template-areas: 'arrow item weight grade duedate';
+`;
+
+const AccordionExtendedComponent = styled.div`
 	border: none;
+	padding-right: 25px;
 `;
 
 const DeleteButton = styled(DelButton)`
-	background-color: #ffffff;
 	color: #b80f0a;
-	margin-right: 2em;
+	width: auto;
+	font-weight: bold;
+	padding-left: 5px;
+	padding-right: 5px;
 `;
 
 function submit() {
 	console.log('changed');
 }
 
+function monthToString(month: number) {
+	switch (month) {
+		case 0:
+			return 'January';
+		case 1:
+			return 'February';
+		case 2:
+			return 'March';
+		case 3:
+			return 'April';
+		case 4:
+			return 'May';
+		case 5:
+			return 'June';
+		case 6:
+			return 'July';
+		case 7:
+			return 'August';
+		case 8:
+			return 'September';
+		case 9:
+			return 'October';
+		case 10:
+			return 'November';
+		case 11:
+			return 'December';
+	}
+}
+
+function dateToString(month: string | undefined, date: number) {
```

Comment:
So I agree with Samantha on using the [`toLocaleDateString()`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date/toLocaleDateString) function because it will consider the users locale. This dateToString works great for us English Canadians, but students who have a different locale set on their computer may expect to see the date differently. 

That being said, if in another project you are going to create a dateToString function I suggest doing it like `dateToString(date: Date)` then calling the other helper methods internally to get the month and the day number within the function. It cleans up the code from the callers perspective and has nice maintainability aspects.

This idea applies to most functions that benefit from a helper function, because then you will only have to call the helper function in one place.